vcd/datasource_vcd_vm_vgpu_policy.go

```diff
@@ -0,0 +1,87 @@
+package vcd
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func datasourceVcdVmVgpuPolicy() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: datasourceVcdVmVgpuPolicyRead,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
Although probably it's pretty obvious, but maybe it's worth it to add a `Description` for completeness (same for `description` attribute).

Reply:
True, fixed.